gcloud/translate/client.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Client for interacting with the Google Cloud Translate API."""
+
+
+import httplib2
+import six
+
+from gcloud._helpers import _to_bytes
+from gcloud.translate.connection import Connection
+
+
+ENGLISH_ISO_639 = 'en'
+"""ISO 639-1 language code for English."""
+
+
+def _zip_assert(values1, values2):
+    """Zips two iterables and asserts they are the same length.
+
+    :type values1: list
+    :param values1: Iterable (with a length) to be zipped.
+
+    :type values2: list
+    :param values2: Iterable (with a length) to be zipped.
+
+    :rtype: iterator
+    :returns: Zip iterator for each pair of values.
+    :raises: :class:`ValueError <exceptions.ValueError>` if the number of
+             values in each list is not the same.
+    """
+    if len(values1) != len(values2):
+        raise ValueError('Expected iterations to have same length',
+                         values1, values2)
+
+    return six.moves.zip(values1, values2)
+
+
+class Client(object):
+    """Client to bundle configuration needed for API requests.
+
+    :type key: string
```

Comment:
So I noticed a mix of `string` vs `str` in other doc strings. Is there a difference or is one more correct than the other?